postgres-example/src/main/java/PGVectorEmbeddingStoreExample.java

```diff
@@ -0,0 +1,53 @@
+
+import org.testcontainers.containers.PostgreSQLContainer;
+import org.testcontainers.utility.DockerImageName;
+import dev.langchain4j.data.embedding.Embedding;
+import dev.langchain4j.data.segment.TextSegment;
+import dev.langchain4j.model.embedding.AllMiniLmL6V2EmbeddingModel;
+import dev.langchain4j.model.embedding.EmbeddingModel;
+import dev.langchain4j.store.embedding.EmbeddingMatch;
+import dev.langchain4j.store.embedding.EmbeddingStore;
+import dev.langchain4j.store.embedding.pgvector.PgVectorEmbeddingStore;
+
+import java.util.List;
+
+public class PGVectorEmbeddingStoreExample {
+
+    public static void main(String[] args) {
+
+        PostgreSQLContainer<?> postgreSQLContainer = new PostgreSQLContainer<>(
+                DockerImageName.parse("ankane/pgvector:v0.5.1").asCompatibleSubstituteFor("postgres"));
```

Comment:
pgvector/pgvector image can be used directly using Testcontainers 1.19.7

Reply:
Thanks @eddumelendez , implemented the change